internal/pkg/coordinator/monitor.go

```diff
@@ -393,21 +396,24 @@ func (m *monitorT) getIPs() ([]string, error) {
 }
 
 func (m *monitorT) rescheduleUnenroller(ctx context.Context, pt *policyT, p *model.Policy) {
-	l := m.log.With().Str(dl.FieldPolicyId, pt.id).Logger()
+	u := uuid.Must(uuid.NewV4())
+	l := m.log.With().Str(dl.FieldPolicyId, pt.id).Str("unenroller_uuid", u.String()).Logger()
```

Comment:
Why the uuid here? Doesn't seem to be used again in the function. The policy ID is already unique.

Reply:
This UUID was used to track goroutines (for debugging the issue).

Reply:
Okay. Still don't know if it's best to keep it, as it's just some random ID. But we can keep it.